CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # PopupBridge iOS Release Notes
 
+## unreleased
+* Bump Xcode version to 15.0.1+
```

Comment:
Is there a reason we need to require the patch `15.0.1` and not just `15.0+`?

Reply:
No particular reason. I'll change to 15.0+ to make it simpler

Reply:
I think we can do `15.0+` same as the BT SDK.

Reply:
The workflows use 15.1. Would there be negative affects if Merchant is using 15.0 instead of 15.1?

Reply:
Our workflow can use a different version than our SDK supports as a minimum version. It seems like Apple is saying `15.0+` is their requirement so that should be ours as well.